frontend/appflowy_flutter/lib/workspace/presentation/home/menu/sidebar/shared/sidebar_new_page_button.dart

```diff
@@ -57,35 +56,28 @@ class _SidebarNewPageButtonState extends State<SidebarNewPageButton> {
   }
 
   Future<void> _createNewPage() async {
-    return createViewAndShowRenameDialogIfNeeded(
-      context,
-      LocaleKeys.newPageText.tr(),
-      (_, __) {
-        // if the workspace is collaborative, create the view in the private section by default.
-        final section =
-            context.read<UserWorkspaceBloc>().state.isCollabWorkspaceOn
-                ? ViewSectionPB.Private
-                : ViewSectionPB.Public;
-        final spaceState = context.read<SpaceBloc>().state;
-        if (spaceState.spaces.isNotEmpty) {
-          context.read<SpaceBloc>().add(
-                const SpaceEvent.createPage(
-                  name: '',
-                  index: 0,
-                  layout: ViewLayoutPB.Document,
-                  openAfterCreate: true,
-                ),
-              );
-        } else {
-          context.read<SidebarSectionsBloc>().add(
-                SidebarSectionsEvent.createRootViewInSection(
-                  name: '',
-                  viewSection: section,
-                  index: 0,
-                ),
-              );
-        }
-      },
-    );
+    // if the workspace is collaborative, create the view in the private section by default.
+    final section = context.read<UserWorkspaceBloc>().state.isCollabWorkspaceOn
+        ? ViewSectionPB.Private
+        : ViewSectionPB.Public;
+    final spaceState = context.read<SpaceBloc>().state;
+    if (spaceState.spaces.isNotEmpty) {
+      context.read<SpaceBloc>().add(
+            const SpaceEvent.createPage(
+              name: '',
+              index: 0,
+              layout: ViewLayoutPB.Document,
```

Comment:
These three could be default values in the event
\nit

Reply:
Making them default values may cause incorrect function calls. I prefer to keep them as required values.